web/app/themes/mitlib-parent/inc/footer-main.php

```diff
@@ -1,23 +1,20 @@
 <?php
 /**
- * The template for the main footer, showing EDS / Barton as discovery options.
+ * The template for the main footer, showing Alma / Primo as discovery options.
  *
- * @package MITlib_Parent
- * @since 0.0.1
+ * @package MIT_Libraries_Parent
+ * @since 1.11.0
  */
 
-namespace Mitlib\Parent;
-
 ?>
 <div class="footer-main flex-container">
 	<div class="links-all flex-container">
 		<div class="flex-item">
 			<h4><a href="/search">Search</a></h4>
-			<a href="/quicksearch" class="link-sub">Quick search</a>
-			<a href="/barton" class="link-sub">Barton catalog</a>
-			<a href="/vera" class="link-sub">Vera: E-journals &amp; databases</a>
+			<a href="/search-collections" class="link-sub">Search Our Collections</a>
 			<a href="/worldcat" class="link-sub">WorldCat</a>
-			<a href="/barton-reserves" class="link-sub">Course reserves</a>
+			<a href="https://libguides.mit.edu/az.php" class="link-sub">Databases A-Z</a>
+			<a href="/search-reserves" class="link-sub">Course reserves</a>
```

Comment:
For the rest of these markup changes, I'm trusting that UX has confirmed that this is what's needed.

Reply:
yup